views/components/HeroCard/HeroCard.css

```diff
@@ -10,12 +10,14 @@
   height: 80px;
 }
 
-@media screen and (min-width: 768px) {
+@media screen and (min-width: 900px) {
   .HeroCard_container {
     width: 565px;
+    padding-top: 162px;
```

Comment:
How do we get to this number? It feels random/too specific but I may just be missing the idea.